homeassistant/components/deconz/config_flow.py

```diff
@@ -159,12 +159,17 @@ def __init__(self):
             title="deCONZ-" + self.deconz_config[CONF_BRIDGEID], data=self.deconz_config
         )
 
-    async def _update_entry(self, entry, host):
+    async def _update_entry(self, entry, host, port, api_key=None):
```

Comment:
This doesn't need to be a coroutine since we don't await inside.

Reply:
Just for my own learning: What is the problem of being a co-routine in that case?

Reply:
It's slower than a callback. There's an overhead to await a coroutine.